packages/blade/src/components/Amount/_decisions/decisions.md

```diff
@@ -0,0 +1,67 @@
+# Amount <!-- omit in toc -->
+
+Amount component is a UI element that displays and formats various currency values in a specified format.
+
+<img  src="./amount-thumbnail.png"  width="380" />
+
+## Design
+
+- [Figma - Amount](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=28012%3A580578&t=3peAz8A2n2Gw4WMl-1)
+
+## API
+
+| Prop                      | Type                                                       | Default     | Description                                                                    | Required |
+| ------------------------- | ---------------------------------------------------------- | ----------- | ------------------------------------------------------------------------------ | -------- |
+| children                  | `number`                                                   | `undefined` | The value to be rendered within the component                                  | ✅       |
+| intent                    | `positive`, `negative`, `notice`, `information`, `blue`    | `undefined` | The variant of the amount to be rendered                                       |          |
+| fontWeight                | `bold`, `regular`                                          | `bold`      | The weight of the amount to be rendered                                        |          |
+| suffix                    | `true`, `false`                                            | `false`     | Indicates whether a suffix should be used                                      |          |
+| size                      | `3xlarge`, `2xlarge`, `xlarge`, `large`, `medium`, `small` | `3xlarge`   | The size of the amount to be rendered                                          |          |
+| isSuffixPrefixHighlighted | `true`, `false`                                            | `true`      | Highlight the main amount by making the prefix symbol and decimal digits small |          |
+
+### Sample Usage
+
+```jsx
+import { Amount } from '@razorpay/components';
+
+<Amount
+  size="2xlarge"
+  isSuffixPrefixHighlighted={true}
+  fontWeight="regular"
+  suffix={false}
+  intent="information"
+>
+  1000.27
```

Comment:
Will have to be `{1000.27}` with proposed API. Added a comment to consider a `value` prop.

Reply:
I think we should use the same approach as other components.

Reply:
Linking to my reasonings [above](https://github.com/razorpay/blade/pull/993#discussion_r1099986273). I feel the same approach wouldn't work here as well because this component is specifically designed to work with numbers and currencies.

Reply:
Sure, let's do that.